test/pluck.js

```diff
@@ -10,13 +10,16 @@ test('pluck', function() {
 
   eq(typeof S.pluck, 'function');
   eq(S.pluck.length, 2);
-  eq(S.pluck.toString(), 'pluck :: Accessible a => String -> Array a -> Array b');
+  eq(S.pluck.toString(), 'pluck :: (Accessible a, Functor f) => String -> f a -> f b');
 
   throws(function() { S.pluck('a', [{a: 1}, {b: 2}]); },
          TypeError,
-         '‘pluck’ expected object at index 1 to have a property named ‘a’; {"b": 2} does not');
+         '‘prop’ expected object to have a property named ‘a’; {"b": 2} does not');
```

Comment:
Could be a bit confusing for users trying to debug an issue and also leaks implementation details.